helpers/corpus.py

```diff
@@ -0,0 +1,89 @@
+import json
+import random
+from shared import config
+
+
+class Corpus:
+    def __init__(self):
+        self.chain = {}
+        self.order = 2
+        self.openingwords = {}
+
+        self.load_json(config['Qud install folder'] +
+                       "/CoQ_Data/StreamingAssets/Base/LibraryCorpus.json")
+
+    def generate_sentence(self, seed="") -> str:
```

Comment:
This could use a comment explaining its general strategy.

Reply:
added several comments explaining stuff https://github.com/TrashMonks/cryptogull/pull/59/commits/e79f1d9647559f62b8cafcfd22e42bb83615f7f2